### DISCUSSION TITLE
How to get ListView's ListItem name as a str  from event

### DISCUSSION THREAD

**Discussion by komidan** [Textualize/textual#3610] [Q&A] [answered]
How do I get the label object's label of the ListItem of the currently selected ListView's ListItem from event
(idk hoping you could understand via code)
```py
def on_list_view_selected(self, event: ListView.Selected) -> None:
    listItemName = ???
```

I've tried things similar to `event.list_view.XXX` and `event.item.name`.

Thanks in advance.

**Comment by davep** [comment, marked as answer]
See this previous discussion: https://github.com/Textualize/textual/discussions/1840
